packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1658,6 +1658,8 @@
   "Report_this_message_question_mark": "Report this message?",
   "Request": "Send Request",
   "Request_to_join_room": "You are not a member of this #<strong>__roomName__</strong> room. Would you like to send request to join",
+  "Request_to_join_room_comment": "A reason, you would like to join this room",
+  "Request_to_join_room_confirm": "This request to join this room is subject to approval. You can add a reason to support this join request",
```

Comment:
This request to join this room is subject to approval. A reason helps the owner of the room to understand why you want to join.

de: Der Eigentümer des Kanals oder ein Moderator muss Deiner Anfrage zustimmen. Wenn Du einen Grund angibst, kann er leichter verstehen, warum Du beitreten möchtest.

Reply:
Translation maintained.